services/sleeper/docker/python/Dockerfile

```diff
@@ -0,0 +1,127 @@
+FROM nvidia/cuda:10.2-base-ubuntu18.04 as base
```

Comment:
ok so since this is not the docker base image that the cookiecutter generated for you, you should use a /custom/ subfolder (see dy-3dvis which does that).
OR
/nvidia/Dockerfile and create a feature request on the cookiecutter as we might have more of these cases in the future. probably first option is ok for now.

docker/custom/Dockerfile or docker/nvidia/Dockerfile

Reply:
I'd put it under the first option